packs/data/pathfinder-bestiary-3.db/thanatotic-titan.json

```diff
@@ -2205,7 +2227,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>The titan can use their Attack of Opportunity when a creature within their reach uses a concentrate action, in additional to its normal trigger. They disrupt actions on any hit, not just a critical hit—including triggering concentrate actions.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.AttackOfOpportunity]</p>"
+                    "value": "<p>The titan can use their Attack of Opportunity when a creature within their reach uses a concentrate action, in additional to its normal trigger. They disrupt actions on any hit, not just a critical hit-including triggering concentrate actions.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.AttackOfOpportunity]</p>"
```

Comment:
```suggestion
                    "value": "<p>The titan can use their Attack of Opportunity when a creature within their reach uses a concentrate action, in additional to its normal trigger. They disrupt actions on any hit, not just a critical hit—including triggering concentrate actions.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.AttackOfOpportunity]</p>"
```
We keep emdash now actually. We removed it before because gitlab refused to show them properly and it made reviewing a pain. I updated the data entry script to not remove them.

Reply:
Ok - thanks for the info :-)